### ISSUE TITLE
What if the "Looking for Maintainers" issue gets closed/reopened?

### ISSUE THREAD

**Issue by flxwu** [flxwu/maintainerswanted.com#26]
1. We could completely delete the project once the issue gets closed, i.e. remove it from the database and deleting the Github webhook. This way, we wouldn't detect the issue reopening and would have to write into the issue description that they should re-register the project on the website if they need more maintainers after they closed the issue.

2. Only delete it from the database, but keeping the WebHook: This way, we would detect the issue reopening and could then add it back into the database. Problem is that this could become quite resource-heavy as the webhook triggers on every change to any issue in the repository.

3. Keep the data in the database and also keep the webhook: I dislike this idea as there is no clearcut-advantage compared to (2.) and it would just unnecessarily use up database resources.

**Comment by Qo2770** [comment]
I would just not support reopening issues at this time. Perhaps comment when the issue is closed that a new issue must be opened in case new maintainers are needed

**Comment by flxwu** [comment]
TL;DR: We write in the issue body that the user will have to re-add the project on the website once the issue is closed, since we are deleting both the DB entry and the webhook if the issue gets closed.
